### PR TITLE
[B] Fix encoding error in url with slugify or remove if not necessary

### PR DESCRIPTION
**Pull request by AlucardleVash**
It's a quick fix for slugify the url.

I think it's possible to have only one slugify function, but i don't know well this php structure

### CODE REVIEW

File: application/views/admin/dashboard/ideas.php [hunk 1]
```diff
@@ -61,7 +60,7 @@
             }
             ?>">
               <td>
-                <a href="<?php echo base_url() . 'home/idea/' . $idea->id . "/" . $freename;?>" target="_blank"><?php echo $idea->title; ?></a>
+                <a href="<?php echo base_url() . 'home/idea/' . $idea->id;?>" target="_blank"><?php echo $idea->title; ?></a>
```
Comment (ivandiazwm) [inline application/views/admin/dashboard/ideas.php]: Be aware that the name of the idea in the url is for SEO purposes. Please don't remove them.
I suggest you add `// TODO: replace this with sluglify functions ` on line 43 and the others $freename, I'll update them later.

**Comment by ivandiazwm** [comment]
It looks good. Thanks!
Just one small comment.

**Comment by AlucardleVash** [comment]
ok, i change this today. I think this link was only for BO propose, so no trouble with SEO. But ok, i try to make something different.

**Comment by AlucardleVash** [comment]
PR up to date. 
I think it's better now. I use Display class with static function slugify.
So one slugify, call everywhere.

**Comment by AlucardleVash** [comment]
is it ok for you ?

**Comment by ivandiazwm** [comment]
Yes, it is. Sorry, I forgot to merge this. Nice job!

**Pull request merged**